eslint.config.mjs

```diff
@@ -46,6 +46,7 @@ export default tseslint.config(
       'linebreak-style': ['error', 'unix'],
       semi: 'warn',
       '@typescript-eslint/ban-ts-comment': 'off',
+      'svelte/no-at-html-tags': 'off',
```

Comment:
I tried to do individual exceptions via comment and something is [broken (again)](https://github.com/sveltejs/eslint-plugin-svelte/issues/355)